helm-charts/batch/Chart.yaml

```diff
@@ -29,7 +29,7 @@ description: |-
   You may obtain a copy of the License at
   
   http://www.apache.org/licenses/LICENSE-2.0
-version: 2.1.0
+version: 2.2.0
```

Comment:
We didn't actually change anything in the Batch chart, so this can be reverted along with the corresponding value in `spm/Chart.yaml`